app/data_access/control_data.py

```diff
@@ -330,3 +351,20 @@ def resolve_observed_infractions(self, info):
             i for i in observed_infractions if i is not None
         ]
         return observed_infractions
+
+    def resolve_can_take_pictures(self, info):
+        if self.control_type != ControlType.lic_papier:
+            return False
+
+        return self.creation_time.date() == datetime.datetime.now().date()
```

Comment:
question: is this boolean only about the current date or should we also include the nb of pictures limitation?

Reply:
I haven't thought about picture limitation yet, it will be done in a separate PR, right now this field is only about allowing adding pictures only for same day controls.